src/cpp/ext/otel/otel_plugin.h

```diff
@@ -336,6 +335,17 @@ class OpenTelemetryPlugin : public grpc_core::StatsPlugin {
   };
 
   // StatsPlugin:
+
+  // Returns the string form of \a key
+  static absl::string_view OptionalLabelKeyToString(
```

Comment:
Do we need this method?  I don't see that we're actually using it anywhere.

Reply:
I use it in `key_value_iterable.h`